packages/acis_thermal_check/test_unit.py

```diff
@@ -0,0 +1,2 @@
+import testr
+testr.testr('acis_thermal_check/tests/dpa', 'acis_thermal_check/tests/acisfp')
```

Comment:
It looks like these args are not limiting to the tests in those two directories.  Am I missing an option or the point?